main.go

```diff
@@ -58,6 +59,7 @@ func init() {
 		fmt.Fprintln(cli.Output())
 	}
 
+	_init.StringVar(&version, "ver", "", "the initial version (defaults to '0.0.0' if not specified)")
```

Comment:
The default version can be specified as the 3rd parameter.

Reply:
> The default version can be specified as the 3rd parameter.

The 3rd parameter expects a string Type, the convention is that the empty string will be evaluated to the semantic Version type of 0.0.0 in the setDefaultVersion function.

Reply:
@adam-intel I see where you are going with this, if I set the default version here to be "0.0.0", I can eliminate the conditional logic in the setDefaultVersion that checks for empty string. We can just have semver.Parse parse the provided or default string accordingly, this will eliminate a few LOC, i like it.